First2.0.Tests.Integration/Integration/HistoricoTest/HistoricoSetup.cs

```diff
@@ -0,0 +1,47 @@
+﻿using First2._0.Application.Models.HistoricoModel;
+using System.Threading.Tasks;
+using Fisrt2._0.Domain.Entidades;
+using Microsoft.EntityFrameworkCore;
+
+namespace First2._0.Tests.Integration.Integration.HistoricoTest
+{
+    public class HistoricoSetup
+    {
+        private static HistoricoSetup _historicoSetup;
+
+        private HistoricoSetup(DbContext dbContext)
+        {
+            Task.FromResult(Seed(dbContext));
+        }
+        public static HistoricoSetup GetSetup(DbContext dbContext)
```

Comment:
Aqui você busca a instancia do objeto, mude a nomenclatura para ser mais coerente
```suggestion
        public static HistoricoSetup GetInstance(DbContext dbContext)
```